std/process.d

```diff
@@ -3127,7 +3127,20 @@ private:
 
             // Cache the last call's result.
             static string lastResult;
-            if (v != lastResult) lastResult = v.idup;
+            if (v.empty)
+            {
+                // dup on a 0-length non-null array returns null rather than
+                // another 0-length non-null arary.  we want to return a
```

Comment:
You can make this comment a lot shorter, it's not public facing.

Note, you spelled "array" wrong.

Something like:

"Return non-null array for blank result to distinguish from not-present result"